apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php

```diff
@@ -282,6 +282,12 @@ function load_wpcom_block_patterns_modifications() {
 	// 	require_once __DIR__ . '/block-patterns/class-block-patterns-modifications.php';
 	// }
 	// phpcs:enable
+	/*
```

Comment:
For when you get back to this PR, PHPCS is complaining that there isn't an empty line above this comment.